src/utils/defaultLanguageStrings.ts

```diff
@@ -316,6 +316,8 @@ export default {
   REVIEW_SUBMISSION_CONFIRM_OPTION: 'Submitting Review',
   REVIEW_SUBMISSION_FAIL: 'Not all sections have been reviewed',
   REVIEW_SUBMISSION_ERROR: 'Something wrong with server - review mutation failed',
+  REVIEW_SUBMISSION_INCOMPLETE_MESSAGE:
+    'You are about to submit your review. Some sections have not be completed by others, which means your decision is not changing the outcome - if aplicable. Are you sure you wish to proceed?',
```

Comment:
Typo on `if applicable`. 

But I'm not actually sure what "if applicable" means in this case.

> `Some sections have not be completed by others,`

Do you mean lower level reviewers? If so, we should probably be specific that we mean that. 

And what happens if they go ahead and submit anyway? I thought we wanted to prevent that altogether.

Reply:
> Do you mean lower level reviewers? If so, we should probably be specific that we mean that.

Well - it could be also other reviewers in the same level - for other sections

Reply:
> And what happens if they go ahead and submit anyway? I thought we wanted to prevent that altogether.

I think preventing would add a little more headache and the need to make a better process for this. For example, if you have 2 reviewers: One for section 1 and another for section 2. Even if the one for section 2 haven't finished yet, you are submitting your sections 1 as APPROVED. The message is only to say - This is a partial submission. It will have to wait until user approves section 2 to move on to the next stage.

Reply:
> It will have to wait until user approves section 2 to move on to the next stage.

> Well - it could be also other reviewers in the same level - for other sections

Okay, I guess I'm just a little confused as to where and how we're checking for this. It seems like two different scenarios:

1. Consolidator is submitting before all lower-level reviews are done -- in which case we should probably prevent this submission completely (if possible?)
2. First reviewer submitting their section with other sections not completed -- warning seems good, but how are we actually checking for this when running the `incrementStage` action in the back-end?

Reply:
> 1. Consolidator is submitting before all lower-level reviews are done -- in which case we should probably prevent this submission completely (if possible?)

No. Users actually liked this option where we allow submitting the consolidation prior to all lower level users have completed. Also because if there are problems if one section it can be fixed before other sections are reviewed.

We also have already implemented logic to block other lower level reviews to be submitted while waiting for applicant to respond. This also prevents many threads of the same review. It should have only one Consolidation done per application per time.

Reply:
> 2\. First reviewer submitting their section with other sections not completed -- warning seems good, but how are we actually checking for this when running the `incrementStage` action in the back-end?

Once this is done - we also need to check on IncrementStage condition if `revieweable_questions_count` = `submitted_questions_count`. The reason why I wanted these 2 to be functions. We might even have a one function in the end to check for that